### DISCUSSION TITLE
Basic settings reference

### DISCUSSION THREAD

**Discussion by uhdotie** [ngoduykhanh/PowerDNS-Admin#1401] [Q&A] [answered]
Is there a reference for the basic settings anywhere? I've search the repo and I've been able to find some explanations in tickets, but a lot of them are hard to understand. I can create a doc similar to the [environment variables reference](https://github.com/PowerDNS-Admin/PowerDNS-Admin/blob/master/docs/wiki/configuration/Environment-variables.md) and fill in what I can, but I wonder if that will end up the same way as that reference, fairly devoid of info. :)

**Comment by AzorianMatt** [comment]
@dahamsta albeit not the most complete or helpful, this has been started some time ago.

Updates to this are certainly a welcome PR. Check this out: https://github.com/PowerDNS-Admin/PowerDNS-Admin/blob/master/docs/wiki/configuration/Environment-variables.md

**Comment by uhdotie** [comment]
I linked to that in my comment, however it's the **Basic Settings** I'm referring to, at /admin/setting/basic in the UI. They don't seem to correlate to the env variables.

**Comment by uhdotie** [comment]
I tried adding an empty template for basic settings however I created it in the wrong place, sorry! Not a big user of GitHub.

**Comment by AzorianMatt** [comment, marked as answer]
@dahamsta my apologies, I completely overlooked your link reference in the original comment. I would expect a near-future update to include in-page documentation of these settings so please be patient and we will hopefully solve that issue. Accordingly, I will plan to provide an associated wiki document to accompany this information as well.

I have created an issue to track this change request and I'll see to it that it gets done for the next major release of 0.4.0.

Thanks!

**Comment by uhdotie** [comment]
Thanks. I'll have another crack off adding a template in a few days.
